quarkus/tests/integration/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM registry.access.redhat.com/ubi8-minimal AS build-env
```

Comment:
Do we still need this file? Aren't you now fetching the docker file via HTTP?

Reply:
As mentioned, I kept the two implementations until a final decision is made.